### PR TITLE
Cheeyong

### PR DESCRIPTION
**Pull request by tchyong**
Update webpack.config.js for standardize naming on visual pack name
Update ReadMe for missing step and better guide explaination

### CODE REVIEW

**Comment by smileEng** [comment]
Please resolve your conflicts first.

File: ReadMe.md [hunk 1]
```diff
@@ -13,10 +13,12 @@
...
 
     - package.json (name, desciption, author)
-    - webpack.config.js ('myCustomVisual' 2 places)
+    - webpack.config.js ('merchantLocationWidget' 2 places to you visual name)
+    - src/program.js ('merchantLocationWidget' to you visual name)
+    - publish/visual/quadrant-properties.js ('merchantLocationWidget' to you visual name)
     - publish/config.js (install.visualConfig)
     - publish/config.js (install.authcookie **GIANT WEB ACCESS COOKIE**)
-    - publish/config.js (install.giantUrl **RELATIVE TO ACCESS COOKIE**)
+    - publish/config.js (install.giantUrl **GIANT URL WHICH YOU GET GIANT WEB ACCESS COOKIE**)
```
Comment (smileEng) [inline ReadMe.md]: Not needed

**Comment by smileEng** [comment]
i would prefer to change the merchantLocationWidget files to my custom widget instead of the other way around

File: ReadMe.md (hunk 1 above)
Comment (tchyong) [reply ReadMe.md]: This is input from Tiat Han, he interpret its as relative URL. 
Hence i update the grammar to more explanation. 
So maintain or change back?

**Comment by tchyong** [comment]
ok..Will update the sample package to "myCustomWidget"

**Pull request merged**